src/workos.spec.ts

```diff
@@ -222,6 +224,30 @@ describe('WorkOS', () => {
       });
     });
 
+    describe('when the api responses with a 429', () => {
+      it('throws a RateLimitExceededException', async () => {
+        fetchOnce(
+          {
+            message: 'Too many requests',
+          },
+          {
+            status: 429,
+            headers: { 'X-Request-ID': 'a-request-id', 'Retry-After': '10' },
```

Comment:
Is 10 seconds the default or is that just for this test? If it's the default returned by the API then that feels way too high.

Reply:
It's just a test value. IIRC, the actual `Retry-After` is generally larger, and varies by endpoint.